container/container.go

```diff
@@ -27,7 +27,8 @@ func NewContainer(containerInfo *types.ContainerJSON, imageInfo *types.ImageInsp
 
 // Container represents a running Docker container.
 type Container struct {
-	Stale bool
+	Stale      bool
+	WasRunning bool
```

Comment:
It would be possible to just use the `IsRunning()` function instead of adding this attribute, because the state is determined during the list call and it's immutable for the container instance. It could only change by retrieving a new updated instance from list containers.